Readme.md

```diff
@@ -0,0 +1,390 @@
+
+   const tracksOnDeckLines = tracksOnDeck.map(trackOnDeck => {
+     return <Line
+       offset={this.state.offset}
+       color={this.props.getColorForTitle(trackOnDeck.title)}
+       key={`${trackOnDeck.title}sec${sectionNum}rank${trackOnDeck.rank}`}
+       weekPosition={sectionNum}
+       y1={this.getPositionForRank(STAGING_AREA_RANK)}
+       y2={this.getPositionForRank(trackOnDeck.rank)}/>;
+   });
+
+   return lines.concat(tracksOnDeckLines);
+ }
+
+ render() {
+   const sectionZero = this.getLinesForSection(0, this.props.chart, this.props.nextChart);
+   const sectionOne = this.getLinesForSection(1, this.props.prevChart, this.props.chart);
+   const sectionTwo = this.getLinesForSection(2, this.props.twoWeeksBackChart, this.props.prevChart);
+   const sectionThree = this.getLinesForSection(3, this.props.threeWeeksBackChart, this.props.twoWeeksBackChart);
+   const sectionFour = this.getLinesForSection(4, this.props.fourWeeksBackChart, this.props.threeWeeksBackChart);
+
+   return (
+     <div id="chart-wrap-wrapper">
+       <div id="chart-wrap">
+         <ul id="chart-y-axis">
+           <li>1 &mdash;</li>
+           <li>2 &mdash;</li>
+           <li>3 &mdash;</li>
+           <li>4 &mdash;</li>
+           <li>5 &mdash;</li>
+           <li>6 &mdash;</li>
+           <li>7 &mdash;</li>
+           <li>8 &mdash;</li>
+           <li>9 &mdash;</li>
+           <li>10 &mdash;</li>
+         </ul>
+         <svg width={700} height={579} style={{ borderBottom: '1px solid white', backgroundColor: 'transparent' }}>
+           {sectionZero}
+           {sectionOne}
+           {sectionTwo}
+           {sectionThree}
+           {sectionFour}
+         </svg>
+       </div>
+       <svg width={700} height={50} style={{ backgroundColor: 'rgb(0, 0, 0)', color: 'white', marginLeft: 'auto' }}>
+         <GraphDate offset={this.state.offset} weekPosition={-1} date={this.props.nextChartDate}/>
+         <GraphDate offset={this.state.offset} weekPosition={0} date={this.props.currentDate}/>
+         <GraphDate offset={this.state.offset} weekPosition={1} date={this.props.prevChartDate}/>
+         <GraphDate offset={this.state.offset} weekPosition={2} date={this.props.twoWeeksBackChartDate}/>
+         <GraphDate offset={this.state.offset} weekPosition={3} date={this.props.threeWeeksBackChartDate}/>
+         <GraphDate offset={this.state.offset} weekPosition={4} date={this.props.fourWeeksBackChartDate}/>
+       </svg>
+     </div>
+   );
+ }
+}
+```
+
+
+The second component is the graph component. this component is in charge of rendering ten album images and track names according to their ranking for a given week. As the rankings change over time, the graph component updates its state which in turn will update positions of current tracks on the graph. Using CSS transitions the tracks will move smoothly towards there new ranking. The graph and charts components work harmoniously together to create a pleasing visualize for top ten tracks.  
```

Comment:
Summarize into one sentence, again just talk high level and let the code do the implementation talking.